elastic/apm-server
# Remove `timeseries.instance` in ingest pipeline for pre-8.6 compatibility

## Motivation/summary



In apmpackage, remove `timeseries.instance` in ingest pipeline so that pre-8.6 apm-server will stop getting ingest error due to strict mapping.

## Checklist




## How to test these changes


- Send txmetrics from a pre-8.6 apm-server (e.g. 8.2) to ES with the latest apm integration package. Ensure there are no errors after ~90 seconds.

## Related issues



Closes #10773
